### PR TITLE
[RF] Homepage (Dashboard)|User is able to add and edit the text in the panel description

### PR DESCRIPTION
**Pull request by IrynaKolyada**
https://trello.com/c/qaRBBcbP/612-rf-homepage-dashboarduser-is-able-to-add-and-edit-the-text-in-the-panel-description

### CODE REVIEW

**Review by ArinaJur — CHANGES_REQUESTED** [review]

File: cypress/e2e/add_description.cy.js [hunk 1]
```diff
@@ -1,10 +1,19 @@
 /// <reference types="cypress" />
 
-describe('AT_02.06_001 | Homepage (Dashboard) > Verify the Main panel description', ()=>{
-    it('Verify the Main panel description', ()=>{
+describe('US_02.06 | Homepage (Dashboard) > Verify the Main panel description', ()=>{
```
[the remark this answers is not in the document]
Comment (IrynaKolyada) [reply cypress/e2e/add_description.cy.js:3]: @ArinaJur Irina I am not sure that I got  what you meant.  I thought I do everything as requested. Please give me an example of another test and i will take a look at it. Thanks a lot.

[the remark this answers is not in the document]
Comment (IrynaKolyada) [reply cypress/e2e/add_description.cy.js]: All done. Please review

[the remark this answers is not in the document]
Comment (ArinaJur) [reply cypress/e2e/add_description.cy.js]: Please check the thread in Slack, and watch yesterday's lesson. 
From Maria's presentation: "describe  содержит в точности название файла"

File: cypress/e2e/add_description.cy.js [hunk 2]
```diff
@@ -1,10 +1,22 @@
 /// <reference types="cypress" />
 
-describe('AT_02.06_001 | Homepage (Dashboard) > Verify the Main panel description', ()=>{
-    it('Verify the Main panel description', ()=>{
+describe('US_02.06 | Homepage Dashboard_Verify the Main panel description', ()=>{
```
Comment (ArinaJur) [inline cypress/e2e/add_description.cy.js:3]: Please change to 
describe ('add_description', () => {

According to yesterday's lesson. We should put the file name to describe

**Review by ArinaJur — CHANGES_REQUESTED** [review]

File: cypress/e2e/add_description.cy.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (IrynaKolyada) [reply cypress/e2e/add_description.cy.js]: @ArinaJur  I misunderstood.  all done . Thank you

File: cypress/e2e/add_description.cy.js (hunk 2 above)
Comment (IrynaKolyada) [reply cypress/e2e/add_description.cy.js]: All done

**Review by ArinaJur — APPROVED** [review]

**Review by Olga-Khliupina — APPROVED** [review]

**Pull request merged**